.proxyrc

```diff
@@ -3,6 +3,6 @@
     "target": "http://localhost:9000/",
   }, 
   "/oauth2": {
-    "target": "http://localhost:9001/"
```

Comment:
Isn't this connected (somehow) to `containerPort: 9001` in `wego-app-dev.yaml`?

Reply:
Possibly, but this `proxy` is meant to be used on conjunction with `tilt`, so we use the port it listens on in the Tiltfile